src/unpackertool/plugins/upx/pe/pe_upx_stub.h

```diff
@@ -9,10 +9,14 @@
 
 #include <unordered_set>
 
+#include <pelib/PeLib.h>
+
 #include "unpackertool/plugins/upx/upx_stub.h"
-#include "retdec/unpacker/dynamic_buffer.h"
+#include "retdec/utils/dynamic_buffer.h"
 #include "retdec/unpacker/signature.h"
 
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.